### ISSUE TITLE
Camps with POOK SAM PACK

### ISSUE THREAD

**Issue by Przemro** [SavageCDN/DAC_A3#11]
How about add pook sam pack to camps. Like he did SAM sites, so this camps could be something like this but created by DAC.

**Comment by SavageCDN** [comment]
Good idea I downloaded the SAM pack but haven't had a chance to look at it.  Will have some time over the holidays to take a look.

**Comment by Przemro** [comment]
okay, I am waiting for CUP update, then I can create units so you will be able to add new armies. (I will try to create 2 version, with POOK SAM and without.

**Comment by SavageCDN** [comment]
Oh yes CUP update will be huge... lots of units to add!!  I'll check out the SAMs this week and see about adding them as DAC camp objects.

**Comment by Delta-99** [comment]
Przemro, any idea when your CUP units will be out? Need something real soon for a mission. Didn't want to re-invent the wheel so came here first to see if anyone is working on such a thing!!

**Comment by Przemro** [comment]
Well I am waiting for CUP to update their mods, now there is only 3 fractions (USMC, German SF and Takistan army). If you need units fast I do it.
